src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -180,7 +180,7 @@ export default {
     },
     resetShapeColor(shape) {
       const defaultColor = this.isPoolShape(shape) ? poolColor : defaultNodeColor;
-      shape.attr('body/fill', defaultColor);
+      shape.attr('body/fill', shape.attr('body/originalFill') || defaultColor);
```

Comment:
I'd love to see this simplified. With this change we have three ways of arriving at the default colour: the pool colour, the default colour or the 'originalFill' property. Could we add an originalFill property to the poolShape? This could then be simplified to just this line:
```suggestion
      shape.attr('body/fill', shape.attr('body/originalFill') || defaultColor);
```
What do you think?